src/components/Fund/FundsList.tsx

```diff
@@ -10,56 +10,128 @@ import {
   ColumnDef,
 } from "@tanstack/react-table";
 
-import { formatAmount, formatDate, USD_DECIMALS } from "../../data/formatting";
+import { formatDate, numberWithCommas } from "../../data/formatting";
 import Table from "../Table/Table";
 import useSWR from "swr";
 import { Fund, FundStatus } from "../../api/models";
 import { api } from "../../config/env";
+import openSeaLogo from "../../img/fundLogos/openSeaLogo.png"
+import upIcon from "../../img/icons/upIcon.svg";
+import downIcon from "../../img/icons/downIcon.svg";
+import Button from "../ui/Button";
+import BorderlessButton from "../ui/BorderlessButton";
+
+
+const GenericColumn = (props: any) => {
+  let returnElement = <div></div>
+  switch (props?.type) {
+    case "status":
+      returnElement = <>
```

Comment:
we dont need returnElement. Just return directly

Reply:
Done